plugin/trino-bigquery/pom.xml

```diff
@@ -256,6 +267,28 @@
             <artifactId>avro</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.13</version>
+            <exclusions>
```

Comment:
What is happening here? Specific version and exclusion looks like it needs a comment :)

Reply:
I'm not depending on classes used transitively by BQ client.

Reply:
Please provide a comment why exclusion is the right thing to do and why it is safe. Same above.